docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+# Simulation Model
+
+## Introduction
+
+Simulation model parameters describe properties of all relevant elements of the observatory including site,
+telescopes, and calibration devices.
+The management, definition, derivation, verification, and validation of the simulation model is central to the functionality of simtools.
+
+Examples for model parameters are the atmospheric models (profile, transmission) or geomagnetic field characteristics for the site,
```

Comment:
I would be more specific on explaining this and would give a couple of examples with parameter names. "parameters are the atmospheric models" seems like we they are not well defined.

Reply:
Added a link to the long list of schema files in the directory (https://github.com/gammasim/simtools/tree/main/simtools/schemas/model_parameters) - this will make it clearer.